### DISCUSSION TITLE
using = and a string special characters (a URL)  - can't figure out the syntax.

### DISCUSSION THREAD

**Discussion by dreaded369** [PostgREST/postgrest#2577] [Q&A] [answered]
### Environment

* PostgreSQL version: 14.5 - docker postgres:
* PostgREST version: - 10.1.0 docker postgrest/postgrest:
* Operating system:

### Description of issue

I'm having trouble figuring out how to find the equivalent of:
SELECT * from items
WHERE urlstring = 'https://example.com"

I've tried:
http://192.168.1.103:3000/items?urlstring=eq.%22https://example.com/%22
http://192.168.1.103:3000/items?urlstring=eq.(%22https://example.com/%22)
http://192.168.1.103:3000/items?urlstring=eq.%22https%3A%2F%2Fexample.com%2F%22
http://192.168.1.103:3000/items?urlstring=eq.(%22https%3A%2F%2Fexample.com%2F%22)
and the same combinations using fts.

What is the method for an exact string search where the contents contain a URL (special characters and .)?

Thanks

**Comment by steve-chavez** [comment, marked as answer]
Have you tried with 

```
http://192.168.1.103:3000/items?urlstring=eq.https://example.com/
```

---

Can confirm it works

```bash
curl 'localhost:3000/pages?url=eq.http://postgrest.org/en/v6.0/api.html'

[{"link":1,"url":"http://postgrest.org/en/v6.0/api.html"}]
```

**Reply by dreaded369** [reply]
huh... yeah that was my first attempt before turning to the docs for url encoding and special characters etc... but I must have had a typo in there somewhere. 

Thanks!
